root - ERROR - error on GET step : ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')) - 2023-04-22 08:02:10,276
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-04-22 08:02:10,276
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-04-22 08:02:10,365
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f634d109d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-22 08:02:15,756
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-22 08:02:15,757
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-22 08:02:15,889
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f16f5c5ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-23 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-23 08:02:14,728
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-23 08:02:14,844
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fcbce4d3d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-24 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-24 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-24 08:02:14,011
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f07206cea90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-25 04:02:13,196
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1cc6d47d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-25 08:02:12,684
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-25 08:02:12,685
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-25 08:02:12,944
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd6dfda9d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-26 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-26 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-26 08:02:13,813
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff0c8430d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-27 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-27 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-27 08:02:12,859
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece84b7d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-28 08:02:15,752
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-28 08:02:15,753
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-28 08:02:22,308
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe4e4566d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-29 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-29 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-29 08:02:15,043
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9c7b49ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-06-30 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-06-30 08:02:13,708
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-06-30 08:02:13,852
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f634c4bdd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-01 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-01 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-01 08:02:15,558
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f00830b7a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-02 04:02:14,220
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe2e76bbd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-02 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-02 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-02 08:02:14,472
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2f9202ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-03 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-03 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-03 08:02:14,985
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fafda299d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-04 08:02:13,709
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-04 08:02:13,710
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-04 08:02:14,116
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f58a09a8d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-05 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-05 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-05 08:02:15,397
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb052338d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-06 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-06 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-06 08:02:14,170
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0f54a97d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-07 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-07 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-07 08:02:14,978
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1022817d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-08 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-08 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-08 08:02:13,819
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5ab0eb1a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-09 04:02:13,192
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff671265d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-09 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-09 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-09 08:02:14,838
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa2dbba9d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-10 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-10 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-10 08:02:13,985
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa7dc4ddd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-11 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-11 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-11 08:02:15,717
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ee2d2ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-12 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-12 08:02:13,708
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-12 08:02:14,240
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f96f7cc4d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-13 08:02:14,729
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-13 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-13 08:02:15,188
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f904ece4d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-14 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-14 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-14 08:02:14,216
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa38223ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-15 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-15 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-15 08:02:12,939
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f25732c4a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-16 04:02:14,216
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f818b3bfd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-16 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-16 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-16 08:02:13,836
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f30cb60fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-17 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-17 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-17 08:02:15,164
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6c00faed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-18 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-18 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-18 08:02:14,058
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc472274d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-19 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-19 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-19 08:02:15,227
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0437b93d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-20 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-20 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-20 08:02:14,166
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3c87f9fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-21 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-21 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-21 08:02:14,962
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2182f6cd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-22 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-22 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-22 08:02:15,154
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0febe31a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-23 04:02:13,192
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f153921fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-23 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-23 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-23 08:02:16,091
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa4a151ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-24 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-24 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-24 08:02:13,801
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbef9313d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-25 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-25 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-25 08:02:15,632
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f607fe23d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-26 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-26 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-26 08:02:13,957
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b2f50fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-27 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-27 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-27 08:02:15,428
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f41f979ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-28 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-28 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-28 08:02:14,512
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2c1d65ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-29 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-29 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-29 08:02:14,806
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f893b773a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-30 04:02:14,217
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb3030f5d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-30 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-30 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-30 08:02:14,101
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff2ef83fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-07-31 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-07-31 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-07-31 08:02:15,081
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7b5c002d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-01 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-01 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-01 08:02:13,935
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f39b5fbfd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-02 08:02:14,729
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-02 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-02 08:02:15,642
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2e9bd7cd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-03 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-03 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-03 08:02:14,217
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb11f569d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-04 08:02:14,730
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-04 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-04 08:02:14,894
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7245e0dd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-05 08:02:13,709
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-05 08:02:13,710
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-05 08:02:15,428
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7582423a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-06 04:02:15,241
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7faa19e94d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-06 08:02:14,729
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-06 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-06 08:02:16,306
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2f6354fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-07 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-07 08:02:13,706
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-07 08:02:14,504
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2e4d7b2d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-08 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-08 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-08 08:02:14,928
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f08ae2d7d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-09 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-09 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-09 08:02:15,615
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5080151d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-10 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-10 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-10 08:02:15,051
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6b8c5fad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-11 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-11 08:02:13,704
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-11 08:02:14,087
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f722a85ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-12 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-12 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-12 08:02:16,193
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ffa049caa90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-13 04:02:14,216
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2d8fcf2d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-13 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-13 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-13 08:02:13,808
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6d42e6ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-14 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-14 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-14 08:02:12,830
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2d02eb6d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-15 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-15 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-15 08:02:13,857
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd1752cad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-16 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-16 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-16 08:02:12,853
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fac9b6fad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-17 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-17 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-17 08:02:13,984
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9f017a1d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-18 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-18 08:02:14,728
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-18 08:02:15,109
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f02c9f8bd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-19 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-19 08:02:13,704
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-19 08:02:13,826
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8161daea90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-20 04:02:13,196
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3be5b69d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-20 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-20 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-20 08:02:16,029
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fec18ba9d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-21 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-21 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-21 08:02:14,070
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f83869a7d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-22 08:02:12,681
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-22 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-22 08:02:13,109
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9d38131d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-23 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-23 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-23 08:02:14,087
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f55eff2cd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-24 08:02:12,681
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-24 08:02:12,684
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-24 08:02:12,831
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4447316d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-25 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-25 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-25 08:02:14,441
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fda05e1fd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-26 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-26 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-26 08:02:12,792
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6b7ebdda90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-27 04:02:14,216
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fca28c7cd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-27 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-27 08:02:13,704
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-27 08:02:13,918
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fcb9885dd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-28 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-28 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-28 08:02:14,850
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f715807cd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-29 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-29 08:02:13,706
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-29 08:02:14,477
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0c9d454d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-30 08:02:12,680
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-30 08:02:12,681
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-30 08:02:12,945
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f657389ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-08-31 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-08-31 08:02:13,706
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-08-31 08:02:13,843
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd646823d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-01 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-01 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-01 08:02:15,118
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9869bddd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-02 08:02:13,705
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-02 08:02:13,706
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-02 08:02:15,262
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8e8449fa90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-03 04:02:13,192
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9556b76d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-03 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-03 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-03 08:02:17,430
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f414607ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-04 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-04 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-04 08:02:13,912
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe9520ebd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-05 08:02:16,777
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-05 08:02:16,778
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-05 08:02:17,379
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f772c699d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-06 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-06 08:02:13,706
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-06 08:02:14,768
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f597f3c1d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-07 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-07 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-07 08:02:14,921
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7eff2f3fcd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-08 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-08 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-08 08:02:13,949
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9c3d912d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-09 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-09 08:02:14,731
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-09 08:02:14,928
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc071ee7a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-10 04:02:14,220
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9eb910ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-10 08:02:15,756
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-10 08:02:15,758
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-10 08:02:21,049
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7faa7103dd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-11 08:02:14,729
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-11 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-11 08:02:15,199
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f603a168d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-12 08:02:15,753
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-12 08:02:15,753
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-12 08:02:15,984
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0e168fed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-13 08:02:14,733
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-13 08:02:14,734
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-13 08:02:15,160
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fcae8612d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-14 08:02:15,757
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-14 08:02:15,757
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-14 08:02:15,947
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8344835d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-15 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-15 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-15 08:02:15,354
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f42cc92bd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-16 08:02:15,756
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-16 08:02:15,756
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-16 08:02:16,510
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f96f98bfa90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-17 04:02:13,196
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f300e7b7d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-17 08:02:14,733
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-17 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-17 08:02:15,954
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f4258f04d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-18 08:02:13,709
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-18 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-18 08:02:14,176
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8a371dad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-19 08:02:14,733
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-19 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-19 08:02:14,841
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9c38e5ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-20 08:02:13,708
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-20 08:02:13,709
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-20 08:02:13,938
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2de36d8d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-21 08:02:14,733
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-21 08:02:14,734
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-21 08:02:16,816
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f037a5add90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-22 08:02:15,752
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-22 08:02:15,753
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-22 08:02:16,085
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fda96695d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-23 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-23 08:02:14,732
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-23 08:02:15,005
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fed9f461a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-24 04:02:14,216
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fee2d958d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-24 08:02:15,757
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-24 08:02:15,757
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-24 08:02:15,995
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa33337ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-25 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-25 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-25 08:02:14,963
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f64f524ad90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-26 08:02:15,752
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-26 08:02:15,755
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-26 08:02:16,072
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb6c37efd90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-27 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-27 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-27 08:02:17,747
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f032c525d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-28 08:02:15,756
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-28 08:02:15,758
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-28 08:02:16,045
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc00e844d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-29 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-29 08:02:14,730
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-29 08:02:14,984
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fefc2ee1d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-09-30 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-09-30 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-09-30 08:02:13,854
root - ERROR - error on POST step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f552d3d1a90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-01 04:02:15,240
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ffa6a9c9d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-01 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-10-01 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-10-01 08:02:14,967
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a2506ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-02 08:02:13,704
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-10-02 08:02:13,705
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-10-02 08:02:13,903
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f5093865d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-03 08:02:14,732
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-10-03 08:02:14,733
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-10-03 08:02:14,899
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fee8086ed90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-04 08:02:15,756
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-10-04 08:02:15,757
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-10-04 08:02:16,286
root - ERROR - error on GET step : HTTPSConnectionPool(host='friends.xn--90aegkbefgpe1bdez5p.xn--p1ai', port=443): Max retries exceeded with url: /api/leads?include=promocodes&filter%5Blanding%5D=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f41ecdf9d90>: Failed to establish a new connection: [Errno 110] Connection timed out')) - 2023-10-05 08:02:14,728
root - ERROR - error at preprocessing step 'NoneType' object is not subscriptable - 2023-10-05 08:02:14,729
root - ERROR - error at compare step  'NoneType' object has no attribute 'query' - 2023-10-05 08:02:14,931
